priv/repo/migrations/20190727140027_create_causes.exs

```diff
@@ -0,0 +1,20 @@
+defmodule SingForNeeds.Repo.Migrations.CreateCauses do
+  @moduledoc """
+    A Cause
+  """
+  use Ecto.Migration
+
+  def change do
+    create table(:causes) do
+      add :name, :string, null: false
+      add :description, :string, null: false
+      add :end_date, :date
+      add :start_date, :date, null: false
+      add :amount_raised, :decimal
+      add :target_amount, :decimal, null: false
```

Comment:
we're sure we don't want to support any use case for causes that don't at first have a description or target amount?